### PR TITLE
fix: increase default cpuRequest for askpass

### PR DESCRIPTION
**Pull request by sdowell**
The askpass sidecar is failing regularly on autopilot due to timing out before the request can be served. This seems to be an issue of insufficient CPU to serve the request within the 1s timeout. The new value seems to pass consistently from experimentation. Intermediate values of 20m and 30m were tested but still flaky.

### CODE REVIEW

File: pkg/reconcilermanager/controllers/reconciler_container_resources.go
Comment (karlkfi) [inline pkg/reconcilermanager/controllers/reconciler_container_resources.go:58]: The problem this fixes is autopilot specific. On standard, the askpass container can boost. So I don't know if we want to increase the requirements for standard to fix autopilot. This is why I wrote https://github.com/GoogleContainerTools/kpt-config-sync/pull/872 to split apart the standard and autopilot resources.

/cc @janetkuo

Comment (nan-yu) [reply pkg/reconcilermanager/controllers/reconciler_container_resources.go:58]: This will fix CI failures and unblock the upcoming 1.16.2 release next week.
I would suggest that we make the change and consolidate with #872 later.

**Review by nan-yu — APPROVED** [review]
/lgtm
/hold

It looks good to me, but I would like @janetkuo to make the final call.

**Pull request merged**
